assests/javascript/index.js

```diff
@@ -1,46 +1,62 @@
+import cityListJson from "./cityList.json" assert { type: "json" };
+
 const apiKey = "8cf248a4fd398e2908cf52a9f187375b";
 const baseUrl = "https://api.openweathermap.org/";
 
 const mainSection = document.getElementById("main");
-const cityNameSelection = document.getElementById("cities");
+const selectedCity = document.getElementById("selected-city");
+const cityNameList = document.getElementById("cities");
 
-cityNameSelection.addEventListener("click", () => {
-  getWeather(cityNameSelection.value);
+selectedCity.addEventListener("keypress", (e) => {
+  if (e.key === "Enter") {
+    getWeather(selectedCity.value);
+  }
 });
 
-getWeather(cityNameSelection.value);
+showDataList();
+getWeather("Rasht");
+
+function showDataList() {
+  return (cityNameList.innerHTML = cityListJson.map((city) => {
+      return `<option>${city.name}</option>`;
+    })
+    .join(""));
+}
 
 async function getWeather(cityName) {
```

Comment:
```suggestion
async function getWeather(cityId) {
```

Reply:
Did it and every detail this change depend on.